src/BattleMode.hpp

```diff
@@ -25,6 +26,7 @@ namespace SokuLib
 		BATTLE_MODE_PRACTICE
 	};
 
+    //! @brief Represents which character you are controlling, or if you are in replay.
```

Comment:
That's untrue. Playing1 and playing2 are both called that way because it's basically 2 different playing states. Iirc there is an actual rule to when it uses 1 or 2 but I don't recall what it is rn. These should just be renamed.
This is the battle sub mode. I don't think there is a better way to describe it.